### PR TITLE
[Testing] Enabling ported UITests from Xamarin.UITests to Appium #2

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

Continuation of https://github.com/dotnet/maui/pull/25178 This is the second group of tests enabled.

### CODE REVIEW

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla25943.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public Bugzilla25943(TestDevice testDevice) : base(testDevice)
 		[Category(UITestCategories.Compatibility)]
 		[FailsOnAndroidWhenRunningOnXamarinUITest]
 		[FailsOnIOSWhenRunningOnXamarinUITest]
-		[FailsOnMacWhenRunningOnXamarinUITest]
+		[FailsOnMacWhenRunningOnXamarinUITestWhenRunningOnXamarinUITestWhenRunningOnXamarinUITest]
```
Comment (PureWeen) [inline src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla25943.cs:25]: :-)

**Review by PureWeen — DISMISSED** [review]

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla25943.cs (hunk 1 above)
Comment (jsuarezruiz) [reply src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla25943.cs:25]: Ooops, fixed.

**Review by jfversluis — CHANGES_REQUESTED** [review]
Looks like we need a rebase here!

**Review by jfversluis — APPROVED** [review]

**Pull request merged**